coul.c

```diff
@@ -1057,6 +1058,7 @@ void report_init(FILE *fp, char *prog) {
     if (clock_is_realtime)
         fprintf(fp, " *RT*");
     fprintf(fp, "\n");
+	fflush(fp);
```

Comment:
This makes sense, but identation is wrong - I suspect you have a tab there.

Reply:
[https://pubs.opengroup.org/onlinepubs/9699919799/functions/fflush.html](https://pubs.opengroup.org/onlinepubs/9699919799/functions/fflush.html)

Reply:
"tab" - like symbol?
Didn't know it was a problem with your code.
Accepted.
I will fix this moment.

Reply:
Oh, I forgot that we have also called `setlinebuf()` on this filehandle, so it is automatically flushed when we write a newline to it. So again, this should not be needed.

`setlinebuf(rfp)` is equivalent to `setvbuf(rfp, NULL, 0, _IOLBF)`, see [setvbuf](https://pubs.opengroup.org/onlinepubs/9699919799/functions/setvbuf.html).